addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -142,6 +144,15 @@ export class HtmlField extends Component {
                         codeViewEl.value = this.props.record.data[this.props.name];
                     }
                 }
+                if (this.props.fullHeight) {
+                    const $editable = this.wysiwyg.getEditable();
+                    const $form_sheet = $editable.closest('.o_form_sheet');
+                    if ($form_sheet.length) {
+                        const { bottom, top } = $editable[0].getBoundingClientRect();
+                        const height = (document.documentElement.clientHeight - top) - ((bottom - $form_sheet[0].clientHeight) * 2 / 3);
```

Comment:
Can you explain this part a little? In your commit message or as a comment in the code :pray: